uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/browser/BrowserEdge.java

```diff
@@ -45,12 +46,30 @@ public class BrowserEdge extends App implements IBrowser {
     private final static String TAG = BrowserEdge.class.getSimpleName();
     private static final String EDGE_PACKAGE_NAME = "com.microsoft.emmx";
     private static final String EDGE_APP_NAME = "Microsoft Edge";
-    private boolean shouldHandleFirstRun = true;
+    private static final String EDGE_APK = "Edge.apk";
+    private boolean shouldHandleAutoFill = true;
```

Comment:
What's the difference between auto fill and first run?

Reply:
Some slight differences in UI elements that edge has based on a "first run" and the specific autofill pop-up that appears. I need to revisit this though, it seems it still not solving the issue.